docker-compose.tpl.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  backend:
```

Comment:
can we call this one `cd`?
I find it confusing that we come up with a different name here

Reply:
Done!